web/src/components/profile/ProfileOpportunitiesAlert.test.tsx

```diff
@@ -0,0 +1,138 @@
+import { ProfileOpportunitiesAlert } from "@/components/profile/ProfileOpportunitiesAlert";
+import { getMergedConfig } from "@/contexts/configContext";
+import { WithStatefulProfileData } from "@/test/mock/withStatefulProfileData";
+import { setupStatefulUserDataContext } from "@/test/mock/withStatefulUserData";
+import {
+  createEmptyProfileData,
+  generateMunicipality,
+  generateProfileData,
+  ProfileData,
+} from "@businessnjgovnavigator/shared";
+import { render, screen, within } from "@testing-library/react";
+
+jest.mock("@/lib/data-hooks/useUserData", () => ({ useUserData: jest.fn() }));
+jest.mock("@/lib/data-hooks/useRoadmap", () => ({ useRoadmap: jest.fn() }));
+
+const Config = getMergedConfig();
+
+describe("<ProfileOpportunitiesAlert />", () => {
+  const renderComponent = (profileData?: ProfileData): void => {
+    render(
+      <WithStatefulProfileData initialData={profileData || createEmptyProfileData()}>
+        <ProfileOpportunitiesAlert />
+      </WithStatefulProfileData>
+    );
+  };
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    setupStatefulUserDataContext();
+  });
+
+  it(
+    "does not display date of formation, home based business, ownership type id, existing employees," +
```

Comment:
[pebble] This test name does not match the outcome. 

1. It says "when those questions are not answered" but they are clearly all answered
2. It says it "does not display [fields]..." but the outcome is actually that it doesn't display the alert at all! This is a big piece of behavior and the test name should clearly reflect that.